app/Http/Controllers/AdminController.php

```diff
@@ -141,7 +138,7 @@ public function upgrade()
         @$Audit = $_POST['Audit'];
         @$ClearAudit = $_POST['Clear'];
 
-        $configFile = $config->get('CDASH_ROOT_DIR') . "/AuditReport.log";
+        $configFile = storage_path("/AuditReport.log");
```

Comment:
This isn't the same path.  The Audit job job writes to `/cdash/app/cdash/AuditReport.log` not `/cdash/storage/AuditReport.log`

I get a 500 when clicking on the button on the maintenance page: 
![image](https://github.com/Kitware/CDash/assets/542106/4a265dd9-6085-4369-9952-3e0d03faf8b9)

Reply:
Fixed.  I'll open a separate PR later to move that file to the storage directory.